pxc-80-backup/Dockerfile

```
@@ -0,0 +1,87 @@
+FROM registry.access.redhat.com/ubi7/ubi-minimal
+
+LABEL name="Percona XtraBackup" \
+	release="2.4" \
+	vendor="Percona" \
+	summary="Percona XtraBackup is an open-source hot backup utility for MySQL - based servers that doesn’t lock your database during the backup" \
+	description="Percona XtraBackup works with MySQL, MariaDB, and Percona Server. It supports completely non-blocking backups of InnoDB, XtraDB, and HailDB storage engines. In addition, it can back up the following storage engines by briefly pausing writes at the end of the backup: MyISAM, Merge, and Archive, including partitioned tables, triggers, and database options." \
+	maintainer="Percona Development <info@percona.com>"
+
+# check repository package signature in secure way
+RUN export GNUPGHOME="$(mktemp -d)" \
+	&& gpg --keyserver pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A 6341AB2753D78A78A7C27BB124C6A8A7F4A80EB5 91E97D7C4A5E96F17F3E888F6A2FAEA2352C64E5 \
```

```suggestion
    && gpg --batch --keyserver pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \
    && gpg --batch --keyserver pool.sks-keyservers.net --recv-keys 6341AB2753D78A78A7C27BB124C6A8A7F4A80EB5 \
    && gpg --batch --keyserver pool.sks-keyservers.net --recv-keys 91E97D7C4A5E96F17F3E888F6A2FAEA2352C64E5 \
```

gpg will only cause this line to fail if either the final key or all keys fail to fetch (can't recall which), which is why we need typically fetch each key on a separate line.

All invocations of gpg should include --batch (see https://bugs.debian.org/913614#27, especially the PS about --batch enabling what's essentially GnuPG's "API mode").